python/cudf/tests/test_gpu_arrow_parser.py

```diff
@@ -158,23 +158,18 @@ def test_gpu_parse_arrow_cats():
         np.testing.assert_almost_equal(got_weight, exp_weight)
 
 
-def make_gpu_parse_arrow_int16_batch():
-    depdelay = np.array([0, 0, -3, -2, 11, 6, -7, -4, 4, -3], dtype=np.int16)
-    arrdelay = np.array([5, -3, 1, -2, 22, 11, -12, -5, 4, -9], dtype=np.int16)
+@pytest.mark.skipif(arrow_version is None,
+                    reason='need compatible pyarrow to generate test data')
+@pytest.mark.parametrize('dtype', [np.int8, np.int16, np.int32, np.int64])
+def test_gpu_parse_arrow_int():
```

Comment:
Looks like you forgot to pass the `dtype` as a parameter here so flake8 complains.

Reply:
```suggestion
def test_gpu_parse_arrow_int(dtype):
```